src/addresses/Signature.ts

```diff
@@ -0,0 +1,41 @@
+import {Wallet} from "..";
+const bchMessage = require('bitcoinjs-message');
+const {randomBytes} = require('crypto');
+
```

Comment:
This is a simple wrapper for randomBytes in nodejs and window.crypto.getRandomValues() elsewhere:

https://github.com/mainnet-cash/mainnet-js/blob/master/src/util/randomBytes.ts

I think in order for the above to work in the browser, a node style 'crypto' has to be added to webpack.